bftengine/src/bftengine/messages/ReplicasRestartReadyProofMsg.cpp

```diff
@@ -13,20 +13,21 @@
 #include "ReplicasRestartReadyProofMsg.hpp"
 #include "SysConsts.hpp"
 #include "EpochManager.hpp"
-#include "SigManager.hpp"
 
 namespace bftEngine {
 namespace impl {
 
 ReplicasRestartReadyProofMsg::ReplicasRestartReadyProofMsg(ReplicaId senderId,
                                                            SeqNum seqNum,
+                                                           RestartReason reason,
```

Comment:
Same as above, if we already introduce a "reason" to the restart proof, then why not have it in the restart message itself and avoid adding another message type?

Reply:
fixed